caravel/viz.py

```diff
@@ -172,12 +172,9 @@ def get_df(self, query_obj=None):
             raise Exception("No data, review your incantations!")
         else:
             if 'timestamp' in df.columns:
-                if timestamp_format == "epoch_s":
+                if timestamp_format in ("epoch_s", "epoch_ms"):
```

Comment:
I feel like the previous approach was more explicit and not relying on magic as much. If someone was to plot historical event in the far past or close to 1970 the magic (or pandas guessing the unit) might not hold as well. 

I'd leave the file as is was if it works.